include/controller/c/webots/utils/ansi_codes.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 1996-2020 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ANSI_CODES_H
+#define ANSI_CODES_H
+
+#include <stdio.h>
+
+#define ANSI_RESET "\x1b[0m"
+
+#define ANSI_BOLD "\x1b[1m"
+#define ANSI_UNDERLINE "\x1b[4m"
+
+#define ANSI_BLACK_BACKGROUND "\x1b[40m"
+#define ANSI_RED_BACKGROUND "\x1b[41m"
+#define ANSI_GREEN_BACKGROUND "\x1b[42m"
+#define ANSI_YELLOW_BACKGROUND "\x1b[43m"
+#define ANSI_BLUE_BACKGROUND "\x1b[44m"
+#define ANSI_MAGENTA_BACKGROUND "\x1b[45m"
+#define ANSI_CYAN_BACKGROUND "\x1b[46m"
+#define ANSI_WHITE_BACKGROUND "\x1b[47m"
+
+#define ANSI_BLACK_FOREGROUND "\x1b[30m"
+#define ANSI_RED_FOREGROUND "\x1b[31m"
+#define ANSI_GREEN_FOREGROUND "\x1b[32m"
+#define ANSI_YELLOW_FOREGROUND "\x1b[33m"
+#define ANSI_BLUE_FOREGROUND "\x1b[34m"
+#define ANSI_MAGENTA_FOREGROUND "\x1b[35m"
+#define ANSI_CYAN_FOREGROUND "\x1b[36m"
+#define ANSI_WHITE_FOREGROUND "\x1b[37m"
+
+#define ANSI_CLEAR_SCREEN "\x1b[2J"
+
+// Convenient macros
+#define ANSI_PRINTF_IN_BLACK(x, ...) printf(ANSI_BLACK_FOREGROUND x ANSI_RESET "\n", ##__VA_ARGS__)
```

Comment:
True it is not the same behavior than printf, however if we don't do this way, the user would systematically need to specify ANSI_RESET along with a `"\n"`.
So an example to just output in Red without the `"\n"` in the macro would be:
`ANSI_PRINTF_IN_RED("Red text!%s\n", ANSI_RESET);`
Otherwise just specifying `"\n"` would color the controller name like here:
![27-03-2020_11:01:28](https://user-images.githubusercontent.com/61777228/77744885-9d600e00-701a-11ea-8293-cdfbd4c61ed2.png)

Reply:
Indeed, this is due to some bad management of the [controller name] prefix with respect to colors... A bug that was here before this PR...

Reply:
I just found it and I believe I can fix it easily. I'll do that on this PR.

Reply:
Okay Olivier, otherwise I would have implemented it since I tested it a little, but if you are on it...

Reply:
Bug fixed, commited there.